qemu/tests/cfg/qmp_event_notification.cfg

```diff
@@ -91,3 +91,11 @@
                 - qmp_none:
                     extra_params += " -watchdog i6300esb -watchdog-action none "
                     action_check = "none"
+        - qmp_guest_panicked:
+            no Windows
+            i440fx,q35:
+              enable_pvpanic = yes
+              ioport_pvpanic = 0x505
```

Comment:
Does the pvpanic device support all platform?

Reply:
@luckyh yes, but it didn't need to enable pvpanic for ppc arch